examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -0,0 +1,66 @@
+"""
+Scatter plot with histograms
+----------------------------
+To create a scatter plot with histograms at the sides of the plot one
+can use :meth:`pygmt.Figure.plot` in combination with
+:meth:`pygmt.Figure.histogram`.
```

Comment:
Maybe mention `fig.shift_origin` as well?

```suggestion
:meth:`pygmt.Figure.histogram`. The positions of the histograms are plotted
by offseting them from the main scatter plot figure using
:meth:`pygmt.Figure.shift_origin`.
```